azure/iac/master-compliance-test.json

```diff
@@ -66,11 +66,47 @@
                         {
                             "cloud": "git",
                             "compliance": [
-                                "CSA-CCM",
+                                "APRA",
+                                "APRA (CPS 234) Information Security-CPS234-23", 
+                                "APRA (CPS 234) Information Security-CPS234-27", 
+                                "APRA (CPS 234) Information Security-CPS234-34", 
+                                "Brazilian Data Protection Law (LGPD)-Article 49", 
+                                "CSA CCM",
+                                "CSA CCM v.4.0.1-AIS-01", 
+                                "CSA CCM v.4.0.1-AIS-02", 
+                                "CSA CCM v.4.0.1-AIS-04", 
+                                "CSA CCM v.4.0.1-CCC-01", 
+                                "CSA CCM v.4.0.1-GRC-03", 
+                                "CSA CCM v.4.0.1-IVS-04", 
+                                "CSA CCM v.4.0.1-UEM-06", 
+                                "CMMC",
+                                "Cybersecurity Maturity Model Certification (CMMC) v.1.02-SC.2.179", 
                                 "HITRUST",
-                                "NIST 800",
-                                "NIST CSF",
-                                "PCI-DSS"
+                                "HITRUST v.9.4.2-Control Reference:10.a", 
+                                "ISO/IEC 27002:2013",
+                                "ISO/IEC 27002:2013-12.1.2", 
+                                "ISO/IEC 27002:2013-14.1.1", 
+                                "ISO/IEC 27002:2013-14.1.2", 
+                                "ISO/IEC 27002:2013-14.2.1", 
+                                "ISO/IEC 27002:2013-14.2.2", 
+                                "ISO/IEC 27017:2015",
+                                "ISO/IEC 27017:2015-12.1.2", 
+                                "ISO/IEC 27017:2015-14.1.1", 
+                                "ISO/IEC 27017:2015-14.1.2", 
+                                "ISO/IEC 27017:2015-14.2.1", 
+                                "ISO/IEC 27017:2015-14.2.5", 
+                                "ISO/IEC 27017:2015-5.1.1", 
+                                "ISO/IEC 27018:2019",
+                                "ISO/IEC 27018:2019-12.1.2", 
+                                "NIST",
+                                "NIST 800-53 Rev 5-Boundary Protection | Connections to Public Networks", 
```

Comment:
we should have a parity between our internal compliance tags, and what we have here.
you should put NIST 800 as an item here as well

Reply:
fixed.